cluster/gce/gci/configure-helper.sh

```diff
@@ -1483,6 +1568,49 @@ function start-etcd-servers {
   prepare-etcd-manifest "-events" "4002" "2381" "100m" "etcd-events.manifest"
 }
 
+# Replaces the variables in the network-proxy manifest file with the real values, and then
+# copy the file to the manifest dir
+# $1: value for variable "cpulimit"
+# $2: value for variable "server_port"
+# $3: value for variable "agent_port"
+# $4: value for bariable "admin_port"
+function prepare-network-proxy-manifest {
+  local -r temp_file="/tmp/network-proxy.manifest"
+  cp "${KUBE_HOME}/kube-manifests/kubernetes/gci-trusty/network-proxy.manifest" "${temp_file}"
+  sed -i -e "s@{{ *cpulimit *}}@$1@g" "${temp_file}"
+  params+=" --log-file=/var/log/network-proxy.log"
+  params+=" --logtostderr=false"
+  params+=" --log-file-max-size=0"
+  params+=" --serverCaCert=${PROXY_SERVER_CA_CERT_PATH}"
+  params+=" --serverCert=${PROXY_SERVER_CERT_PATH}"
+  params+=" --serverKey=${PROXY_SERVER_KEY_PATH}"
+  params+=" --clusterCaCert=${PROXY_AGENT_CA_CERT_PATH}"
```

Comment:
Where do these flag names come from? The convention is lowercase with - seperator.

Reply:
Fixed